.rhcicd/clowdapp-aggregator.yaml

```diff
@@ -47,6 +44,26 @@ objects:
         volumeMounts:
         - name: rds-client-ca
           mountPath: /tmp
+        readinessProbe:
+          httpGet:
+            path: /health/ready
+            port: 8000
+            scheme: HTTP
+          initialDelaySeconds: 1
+          periodSeconds: 1
+          timeoutSeconds: 1
+          successThreshold: 1
+          failureThreshold: 3
```

Comment:
I'm not sure i understand, but are these considering the usual time the cronjob takes to execute? 
Won't these affect the running process?

Reply:
the timing is so that it succeeds near instantly, so the cronjob will be ready and healthy.  It's a workaround for kubelinter to not alert on it. Shouldn't affect runtime, and can be reverted when the above clowder change rolls out